stellargraph/core/graph.py

```diff
@@ -842,7 +842,9 @@ def node_degrees(self) -> Mapping[Any, int]:
         """
         return self._edges.degrees()
 
-    def to_adjacency_matrix(self, nodes: Optional[Iterable] = None):
+    def to_adjacency_matrix(
+        self, nodes: Optional[Iterable] = None, ignore_weights=True
```

Comment:
I personally find a positive phrasing clearer; that is, changing from the default is turning something on, rather than turning something off.

```suggestion
        self, nodes: Optional[Iterable] = None, weighted=False
```

and then `weighted=True` would use the weights from the graph. But I guess that makes the multi-edge explanation a bit more awkward?

(Spelling out an example here for my own exploration, you don't have to use.)


```python
        """
        Obtains a SciPy sparse adjacency matrix of edge weights.

        By default (``weighted=False``), each element of the matrix contains the number 
        of edges between the two vertices (only 0 or 1 in a graph without multi-edges).

        Args:
            nodes (iterable): The optional collection of nodes
                comprising the subgraph. If specified, then the
                adjacency matrix is computed for the subgraph;
                otherwise, it is computed for the full graph.
            weighted (bool): If true, use the edge weight column from the graph instead
                of edge counts (weights from multi-edges are summed).

        Returns:
             The weighted adjacency matrix.
        """
```

Reply:
Thanks, I used your example I think the multi-edge explanation sounds fine, and not far from "expected" behaviour.